sql/src/test/java/io/crate/analyze/CreateAlterTableStatementAnalyzerTest.java

```diff
@@ -186,6 +194,20 @@ public void testAlterTableWithRefreshInterval() throws Exception {
     }
 
     @Test
+    public void testAlterTableWithTotalFieldsLimit() throws Exception {
+        AlterTableAnalyzedStatement analysisSet = e.analyze(
```

Comment:
I am not sure if that is necessary.
It's one thing to test the default values of the setting (definition), another to test that it's correctly wired up with alter/create table. But I'm not sure if it makes sense to start testing all combinations of settings/create/alter table statements?